src/tests/Stream/PartitionTests.cpp

```diff
@@ -58,8 +53,8 @@ class PartitionStandIn : public Stream::Partition {
                    Stream::time_point Stop, Stream::duration StopLeeway,
                    Stream::duration KafkaErrorTimeout)
       : Stream::Partition(std::move(Consumer), Partition, std::move(TopicName),
-                          Map, Writer, RegisterMetric, Start, Stop, StopLeeway,
-                          KafkaErrorTimeout) {}
+                          Map, Writer, std::move(RegisterMetric), Start, Stop,
```

Comment:
I am pretty sure that `std::move` is a no-op in this case and thus useless.

Reply:
Doesn't look like it to me, and `clang-tidy` suggested the change and usually gets that one right.